src/test/java/edu/cornell/kfs/concur/businessobjects/ValidationResultTest.java

```diff
@@ -0,0 +1,54 @@
+package edu.cornell.kfs.concur.businessobjects;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ValidationResultTest {
+    
+    private ValidationResult validationResult;
+    
+    private static final String TEST_STRING_ONE = "test string 1";
+    private static final String TEST_STRING_TWO = "test string 2";
+    private static final String TEST_STRING_THREE = "test string 3";
+
+    @Before
+    public void setUp() throws Exception {
+        validationResult = new ValidationResult();
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        validationResult = null;
+    }
+
+    @Test
+    public void verifyAddMessage() {
+        validationResult.addMessage(TEST_STRING_ONE);
+        validationResult.addMessage(TEST_STRING_TWO);
+        validationResult.addMessage(TEST_STRING_ONE);
+        validationResult.addMessage(TEST_STRING_THREE);
+        validationResult.addMessage(TEST_STRING_TWO);
+        assertEquals("There should only be three lines", 3, validationResult.getMessages().size());
+    }
+    
+    @Test
+    public void verifyAddMessages() {
+        List<String> messagesToAdd = new ArrayList();
```

Comment:
You still need at least the empty "<>" on the other one, so that the compiler knows to derive the generic type information from the left-hand side. Otherwise, the right-hand side will just be using a raw type.